src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -143,9 +146,13 @@ const Body = BaseForm.extend(Object.assign(
 
     doCustomURI () {
       this.ulDom && this.ulDom.remove();
-      this.ulDom = this.add(`
-        <iframe src="${this.customURI}" id="custom-uri-container" style="display:none;"></iframe>
-      `).last();
+      this.ulDom = this.add(View.extend({
+        template: hbs`<iframe src="{{{customURI}}}" id="custom-uri-container" style="display:none;"></iframe>`
```

Comment:
just use the pure string as before? or does the `template` have to be handlerbar template now?

Reply:
At this point in time, we are allowing `add()` with a string. As long as it doesn't have handlebars in it, there is technically no problem with that. Before I added the karma enforce precompile check, I was converting some of these cases to be Views with templates because I had a warning for using `add()` with a bare string. In this case we can probably go back to pure string.